cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/AndroidConfig.kt

```diff
@@ -1,18 +1,25 @@
 package ftl.config
 
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.core.type.TypeReference
 import com.linkedin.dex.parser.DexParser
 import ftl.android.*
 import ftl.gc.GcStorage
 import kotlinx.coroutines.experimental.runBlocking
 
 class AndroidConfig(
+        @field:JsonProperty("app")
         val appApk: String = "",
+        @field:JsonProperty("test")
         val testApk: String = "",
+        @field:JsonProperty("auto-google-login")
         val autoGoogleLogin: Boolean = true,
+        @field:JsonProperty("use-orchestrator")
         val useOrchestrator: Boolean = true,
```

Comment:
this means that `useOrchestrator` is on by default?

Reply:
> this means that useOrchestrator is on by default?

Yes, this is the recommended way to run espresso tests with Flank.